users/views.py

```diff
@@ -6,15 +6,30 @@
 from rest_framework.response import Response
 from rest_framework_simplejwt.views import TokenObtainPairView
 
+from .permissions import IsOwnerOrReadOnly
 from .serializers import (
     PasswordResetRequestSerializer,
     PasswordResetSerializer,
+    UserSerializer,
     UserTokenObtainPairSerializer,
 )
 
 User = get_user_model()
 
 
+class UserList(generics.ListCreateAPIView):
+    permission_classes = (IsOwnerOrReadOnly,)
```

Comment:
How can a new user register? this permission will deny them, also you are allowing anybody to just view all the users, don't you think this is a security risk?